src/styles/styles.js

```diff
@@ -347,19 +347,27 @@ const styles = {
         width: variables.componentSizeNormal,
     },
 
-    textInput: {
+    textInputContainer: {
+        flex: 1,
         backgroundColor: themeColors.componentBG,
         borderRadius: variables.componentBorderRadius,
-        height: variables.componentSizeNormal,
         borderColor: themeColors.border,
         borderWidth: 1,
-        color: themeColors.text,
-        fontFamily: fontFamily.GTA,
-        fontSize: variables.fontSizeNormal,
         paddingLeft: 12,
         paddingRight: 12,
         paddingTop: 10,
         paddingBottom: 10,
+    },
+    textInput: {
```

Comment:
Heads up, this style is used in more than one place so when you modify you must check all other usages to be certain the change is safe.

Reply:
I'm not sure if this helps, but one idea is that everything which uses a `textInput` may also require a `textInputContainer` around it now like the one we've added to the sidebar input.

Reply:
As the bug was reproducible only at the single place, I have created separate styling objects for the `Chat Switcher`. I have pushed another commit. Now, it will not affect the other places. I am not able to test the build on the web, but I have also added the `borderColor` to the `TextInput` to be `themeColors.componentBG` as per my understanding to remove the black border on the web. Please review the latest commit to compare the changes.

Reply:
>I have also added the borderColor to the TextInput to be themeColors.componentBG as per my understanding to remove the black border on the web

That is not an acceptable solution as the black border is required and this issue has not asked for it to be removed.